docs/howto/shared-cache.rst

```diff
@@ -0,0 +1,73 @@
+.. _howto-shared-cache:
+
+Cache intermediate build artefacts
+==================================
+
+Because Charmcraft builds Python packages from source rather than using pre-built
+wheels, the initial builds of charms can take a while. However, future builds of
```

Comment:
> can take a while

To really sell the benefits of this task, we should probably share our opinion and not kneecap ourselves. Really, we're saying:

> initial builds are characteristically slow, _and_ it's best not to repeat them.

Reply:
It's unclear right now whether we want to make this the preferred way to do it, which is why I'm hedging here.

Reply:
How about:

> However, future builds of the same charm are sped up significantly...

↓

> You can significantly speed up future builds of the same charm...

Reply:
That works, too! Noting here that there's no ban on the word "you" in how-tos. It's perfectly fine to use it.